cvescan/constants.py

```diff
@@ -105,3 +105,12 @@
 NULL_LOGGER_NAME = "cvescan.null"
 STDOUT_LOGGER_NAME = "cvescan.stdout"
 SYSLOG_LOGGER_NAME = "cvescan.syslog"
+
+SUPPORTED_RELEASES = [
+    "trusty",
+    "xenial",
+    "bionic",
+    "eoan",
```

Comment:
eoan aka Ubuntu 19.10 has reached end of supported status, and so probably shouldn't be included here.

Reply:
Good catch. I'll push a fix.

Reply:
Fixed in 40fe0b7bb8496eee76079823c5ed09e9100bd106